test/integration/elements/test-element.ts

```diff
@@ -9,7 +9,7 @@
  * rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
-/// <reference path="../bower_components/polymer-decorators/global.d.ts" />
+/// <reference path="bower_components/polymer-decorators/global.d.ts" />
```

Comment:
your build fails because this path can't be found, probably just needs setting back to `../`?